plugins/fuse/server/server_fuse.py

```diff
@@ -0,0 +1,492 @@
+import atexit
+import errno
+import fuse
+import os
+import shutil
+import six
+import stat
+import sys
+import subprocess
+import threading
+import time
+
+from girder import events, logger, logprint
+from girder.constants import AccessType
+from girder.exceptions import AccessException, ValidationException
+from girder.models.file import File
+from girder.models.folder import Folder
+from girder.models.item import Item
+from girder.utility.model_importer import ModelImporter
+from girder.utility import path as path_util
+
+
+# There can be multiple FUSE mounts, each with different user permissions.  If
+# the mount needs to be exposed in some manner to something other than the
+# main server, this could be used to limit which resources are visible.  Since
+# the server can already access all resources, this is not of utility from
+# within the server itself.
+fuseMounts = {}
+fuseMountsLock = threading.RLock()
+
```

Comment:
Should fuseMounts and fuseMountsLock be public symbols here?  Are they intended to be used outside the context of this module/plugin?

Reply:
No.  I'll prefix them with _ to mark them private.